### PR TITLE
fix xauth configurations

### PR DESCRIPTION
**Pull request by asakapab0i**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix
- [ ] New feature

### Testing

- [ ] Testing information has been added - test cases checklist and steps followed for testing.
- [ ] Testing screenshot(s) attached for more information.

## Security

- [ ] This PR has security related considerations.

### CODE REVIEW

File: scripts/config.py [hunk 1]
```diff
@@ -179,22 +179,12 @@ def gather_user_data_s3(s3_url):
 
     config = json.loads(data)
 
-    # if it has 'local_cidr', then use that value for l2tp_cidr
```
Comment (aren) [inline scripts/config.py]: This is going to invalidate config files that our customers have stored in s3. We'll need to come up with a way to communicate that this build isn't backwards compatible.

File: scripts/config.py [hunk 2]
```diff
@@ -179,22 +179,12 @@ def gather_user_data_s3(s3_url):
...
-        config['l2tp_cidr'] = local_cidr
-
     return config
 
 
 def gather_user_data_file(filename):
     config = json.load(open(filename))
 
-    # if it has 'local_cidr', then use that value for l2tp_cidr
-    local_cidr = config.pop('local_cidr', None)
```
Comment (aren) [inline scripts/config.py]: Same feedback as above

**Review by aren — COMMENTED** [review]
I don't see anything here that indicates xauth was broken. what am I missing?

File: scripts/config.py (hunk 1 above)
Comment (asakapab0i) [reply scripts/config.py]: Hi @aren , I think we can just revert this, the issue here is that our example json content is outdated.

**Review by aren — APPROVED** [review]

**Pull request merged**